.github/workflows/docs-ci.yml

```diff
@@ -0,0 +1,29 @@
+name: Docs CI
+on:
+  push:
+    branches: [main]
+    paths:
+    - 'docs/**'
+    - '!docs/**/*.md'
+  pull_request:
+    branches: [main]
+    paths:
+    - 'docs/**'
+    - '!docs/**/*.md'
+
+jobs:
+  docs_ci:
+    name: Verify Files
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+    - name: Install Docs Packages
+      working-directory: docs
+      run: npm install
+    - name: Stylelint Docs
+      working-directory: docs
+      run: npm run lint:scss
+    - name: Lint Docs JS Files
+      run: node Makefile lintDocsJS
```

Comment:
This [runs eslint](https://github.com/eslint/eslint/blob/337f7ed96131d873be7ae6b010739476d0ad15e9/Makefile.js#L560), so we need to do `npm install` in the project root too, to install eslint's dependencies.